src/components/Comment/Comment.tsx

```diff
@@ -31,6 +32,7 @@ const commentControlsButton = (pillar: Pillar) => css`
   ${textSans.xsmall({ fontWeight: "bold" })}
   margin-right: ${space[2]}px;
   color: ${palette[pillar][400]};
+  background-color: ${neutral[100]};
```

Comment:
The correct background colour is `background.primary` - we don't need to import `neutral` here